### ISSUE TITLE
Bootsy doesn't start when adding jQuery via Yarn.

### ISSUE THREAD

**Issue by gafemoyano** [volmer/bootsy#272]
Hi. First of all thank you for bootsy, it's a great editor that does exactly what it needs to do.

So I was trying to add bootsy to a Rails 5.1.RC1 project and I couln't get it to show me the editor buttons.
This was very weird since the *.js and *.css files where getting included in the bundle and the text areas were getting rendered with all the data-bootsy-* attributes. The web console also didn't show any errors on Chrome, Firefox, or IE.
Anyways after digging for a while I figured that the only thing weird thing I was doind was requiring jQuery from the npm_modules folder. As you probably know, rails 5.1 will include yarn by default, so I decided to install front end dependencies that way.

Turns out that by adding back `gem 'jquery-rails'`  to the Gemfile solved the issue and bootsy displayed correctly after it.

I'm not sure if this is an issue with bootsy itself, but it would be nice to at least have a warning somewhere (maybe the docs?) indicating that bootsy needs the jquery-rails gem to work. 

I'd be happy to create a pull request if you point me in the right direction.

**Comment by volmer** [comment]
Hey

I didn't start updating Bootsy to Rails 5.1 yet but it shouldn't require `jquery-rails` specifically. As long as you have jQuery required before Bootsy's assets everything should work fine.

**Comment by gafemoyano** [comment]
Yeah I know it shouldn't but it doesn't work if you include jQuery from node modules. Maybe it's a rails thing? Here's a reproduction repo [](https://github.com/gafemoyano/yarn-bootsy-test) if you want to check it out when you got time. Rember to run `yarn install` to download jQuery. 
There's a bootsy_area at [](http://localhost:3000/posts/new)

**Comment by volmer** [comment]
The problem in the repo you mentioned is that your SCSS manifest is named `application.css`. In order to use SCSS's `@import` statements you need to rename that file to `application.scss` and restart your server. Bootsy then will work.

**Comment by gafemoyano** [comment]
Sorry about that, I forgot to rename the file when I created the demo. 
But I'm pretty sure that's not it I renamed it and checked that the sources were being imported properly just in case and it still doesn't work. 
![screenshot](https://cloud.githubusercontent.com/assets/1516349/25240849/03aba6d4-25c2-11e7-86e5-ead79929516f.jpg)

As I said, the fix that worked for me was to include jquery-rails in the Gemfile for it to work.

Cheers.
